.circleci/config.yml

```
@@ -32,6 +32,40 @@ jobs:
             command: |
               pip install coveralls
               coveralls
+        - run:
+            name: test validate files and yaml
+            command: |
+              . .tox/py37/bin/activate
+
+              git clone https://github.com/demisto/content.git
+              cd content
+
+              if [[ $CIRCLE_BRANCH != master && $FAKE_MASTER != true ]]; then
```

what's `FAKE_MASTER` supposed to mean?

---

Like we talked today, it is a way to trigger the sdk build as it would run on master

---

in that case, you can switch the condition to depend only on that variable (when not defined, will enter the condition scope):
```suggestion
              if [[ -z "$FAKE_MASTER" ]]; then
```